services/identity/ReposService.ts

```diff
@@ -34,14 +35,15 @@ type repoCreateParamsType = Partial<Repo> & {
   siteId: Repo["siteId"]
 }
 export default class ReposService {
-  // NOTE: Explicitly specifying using keyed properties to ensure
-  // that the types are synced.
-  private readonly repository: ReposServiceProps["repository"]
+  // NOTE: No need to explicitly specify types if they are assigned from props in constructor.
+  private readonly repository
 
   constructor({ repository }: ReposServiceProps) {
     this.repository = repository
   }
 
+  localRepoPath = (repoName: string) => `/tmp/${repoName}`
+
```

Comment:
Nit: could we have a more descriptive `getLocalRepoPath`?

Reply:
That is better. I'll make the change.